pkg/apis/duck/v1alpha1/channel_defaulter.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// TODO This should be placed in channel_defaults.go within messaging once Subscription is moved to messaging.
```

Comment:
This seems like worthy of an issue to track this work?

Reply:
Yes! [Issue filed](https://github.com/knative/eventing/issues/1562)
Thanks!